### PR TITLE
IFluidResolvedUrl Deprecated

### PR DESCRIPTION
**Pull request by anthony-murphy**
### IFluidResolvedUrl Deprecated

IFluidResolvedUrl is now deprecated, all usages should move to IResolvedUrl instead. This aligns with the preceding change, [IResolvedUrl equivalent to IFluidResolvedUrl](https://github.com/microsoft/FluidFramework/blob/next/BREAKING.md#IResolvedUrl-equivalent-to-IFluidResolvedUrl)

### CODE REVIEW

File: api-report/driver-definitions.api.md [hunk 1]
```diff
@@ -247,7 +233,21 @@ export interface ILocationRedirectionError extends IDriverErrorBase {
...
+    id: string;
+    // (undocumented)
+    tokens: {
+        [name: string]: string;
+    };
+    // (undocumented)
+    type: "fluid";
+    // (undocumented)
+    url: string;
+}
```
Comment (ChumpChief) [inline api-report/driver-definitions.api.md:250]: Would be nice if you have time to add documentation while you're here :)

File: BREAKING.md [hunk 2]
```diff
@@ -19,6 +19,12 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 
 ## 2.0.0-internal.5.0.0 Upcoming changes
 
+-   [IFluidResolvedUrl Deprecated](#IFluidResolvedUrl-Deprecated)
+
+### IFluidResolvedUrl Deprecated
+
+IFluidResolvedUrl is now deprecated, all usages should move to IResolvedUrl instead. This aligns with the preceding change, [IResolvedUrl equivalent to IFluidResolvedUrl](#IResolvedUrl-equivalent-to-IFluidResolvedUrl)
```
Comment (ChumpChief) [inline BREAKING.md:26]: In addition to the upcoming deprecation, you're changing various public API signatures in this PR to use/return IResolvedUrl - should update the "IResolvedUrl equivalent to IFluidResolvedUrl" breaking change to note this.

File: api-report/container-loader.api.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ export interface IContainerLoadOptions {
     clientDetailsOverride?: IClientDetails;
     loadMode?: IContainerLoadMode;
     // (undocumented)
-    resolvedUrl: IFluidResolvedUrl;
+    resolvedUrl: IResolvedUrl;
```
Comment (anthony-murphy) [inline api-report/container-loader.api.md:60]: this is deprecated in main, and will be removed from public exports in branch in an upcoming PR

File: BREAKING.md (hunk 2 above)
Comment (anthony-murphy) [reply BREAKING.md:26]: this change no longer updates public signatures. It was safe to do, but better to do in the later removal PR.

**Review by ChumpChief — APPROVED** [review]

**Pull request merged**